CONTRIBUTING.md

```diff
@@ -0,0 +1,179 @@
+# Contribution Guidelines
+
+If you are interested in contributing to Substrate Documentation, please follow these contribution 
+guidelines.
+
+* [PR Checklist](#pr-checklist)
+* [Documentation Standards](#documentation-standards)
+* [Documentation Style](#documentation-style)
+
+## PR Checklist
+
+- [ ] Are the audience and objective of the document clear? E.g. a document for developers that 
+  should teach them about transaction fees.
+- [ ] Is the writing:
+  - **Clear**: No jargon
+  - **Precise**: No ambiguous meanings
+  - **Concise**: Free of superfluous detail
+- [ ] Does it follow our [style guide](#documentation-style)?
+- [ ] If this is a new page, does the PR include the appropriate infrastructure, e.g. adding the
+  page to a sidebar?
+- [ ] Build the page. Does it render properly? E.g. no funny lists or formatting.
+- [ ] Do links go to rustdocs or devhub articles rather than code?
```

Comment:
is this good or bad if they do? The wording makes it unclear which is preferred...

Reply:
Breaking my own precision rules! Thanks.